app/components/NewCredentialForm.tsx

```diff
@@ -0,0 +1,159 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import type { DataFunctionArgs } from '@remix-run/node'
+import { redirect } from '@remix-run/node'
+import { Form, Link, useLoaderData } from '@remix-run/react'
+import { Button, Dropdown, Label, TextInput } from '@trussworks/react-uswds'
+import { useState } from 'react'
+import ReCAPTCHA from 'react-google-recaptcha'
+import { getEnvOrDieInProduction } from '~/lib/env'
+import { ClientCredentialVendingMachine } from '~/routes/user/client_credentials.server'
+
+export async function loader({ request }: DataFunctionArgs) {
+  return await handleCredentialLoader({ request } as DataFunctionArgs)
+}
+
```

Comment:
Remove; this is dead code.

```suggestion
```

Reply:
The loader function is referenced later in the file, the returned group and the recaptcha key